packages/charts/src/components/legend/legend_icon.tsx

```diff
@@ -9,36 +9,43 @@
 import React from 'react';
 
 import { ShapeRendererFn } from '../../chart_types/xy_chart/renderer/shapes_paths';
-import { Color } from '../../utils/common';
-import { PointStyle } from '../../utils/themes/theme';
-import { Icon } from '../icons/icon';
+import { Color, isNil } from '../../utils/common';
+import { PointShape, PointStyle } from '../../utils/themes/theme';
 
 interface LegendIconProps {
   pointStyle?: PointStyle;
   color: Color;
 }
 
-const MARKER_SIZE = 16;
+const MARKER_SIZE = 8;
+
+// to limit size, set a min and max
```

Comment:
Let's make this a tsdocs comment

```suggestion
/** to limit size, set a min and max */
```

Reply:
80c4cea thanks!